### PR TITLE
add clearer messaging for Spring Zeebe

### PR DESCRIPTION
**Pull request by akeller**
## Description

From internal thread - https://camunda.slack.com/archives/C06MPP9DQ21/p1715891578979259?thread_ts=1715156875.533999&cid=C06MPP9DQ21

This change is intentionally only backported to 8.5+.

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [x] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [x] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer.

### CODE REVIEW

**Review by christinaausley — APPROVED** [review]

File: versioned_docs/version-8.5/components/best-practices/development/writing-good-workers.md [hunk 1]
```diff
@@ -126,7 +130,11 @@ ZeebeClient client = ZeebeClient.newClientBuilder()
   .build();
 ```
 
-Or, in Spring Zeebe:
+:::caution
+[Spring Zeebe](https://github.com/camunda-community-hub/spring-zeebe) is a community-maintained project.
+:::
```
Comment (saig0) [inline versioned_docs/version-8.5/components/best-practices/development/writing-good-workers.md:135]: :wrench: We have the same warning just a few lines above. We could drop the second warning here.

**Review by saig0 — APPROVED** [review]
@akeller looks good. :+1: I added one suggestion. Please have a look. :cookie:

File: versioned_docs/version-8.5/components/best-practices/development/writing-good-workers.md (hunk 1 above)
Comment (akeller) [reply versioned_docs/version-8.5/components/best-practices/development/writing-good-workers.md:135]: Much like 🍪s I love ⚠️s. I will drop it 😆

**Review by saig0 — APPROVED** [review]
@akeller LGTM. :rocket:

**Review by christinaausley — APPROVED** [review]

**Pull request merged**